lti_emailer/canvas_api_client.py

```diff
@@ -107,7 +131,3 @@ def _get_authorized_teaching_staff_enrollment_types(account_id):
             role_name, account_id, canvas_api_helper_courses.COURSE_PERMISSION_SEND_MESSAGES_ALL
         )
     ]
-
-
-def _copy_user_attributes_to_enrollment(user, enrollment):
-    enrollment.update({a: user[a] for a in USER_ATTRIBUTES_TO_COPY})
```

Comment:
Some of these methods just moved around? It was a bit confusing..I am presuming get_courses_for_account_in_term  is new but that none of the other methods have any logic change?